core/toolbox.js

```diff
@@ -37,6 +39,9 @@ goog.requireType('Blockly.IBlocklyActionable');
  *     blocks.
  * @constructor
  * @implements {Blockly.IBlocklyActionable}
+ * @implements {Blockly.IDeleteArea}
+ * @implements {Blockly.IRegistrable}
```

Comment:
`Blockly.IRegistrable` is redundant here, leave out. Just implements `Blockly.IToolbox`